linux/src/platform_linux.c

```diff
@@ -1,13 +1,53 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#include "c_logging/xlogging.h"
+
 #include "c_pal/platform.h"
```

Comment:
>"c_pal/platform.h" [](http://example.com/codeflow?start=9&length=18)

Is this intended to be "platform_linux.h"? Seeing as that's where `platform_get_completion_port` is declared.